crates/ide-completion/src/completions/postfix.rs

```diff
@@ -516,6 +526,44 @@ fn main() { 42.$0 }
 use core::ops::ControlFlow;
 
 fn main() { ControlFlow::Break(42) }
+"#,
+        );
+
+        check_edit_with_config(
+            config.clone(),
+            "break",
+            r#"
+//- minicore: try
+fn main() { '\\'.$0 }
+"#,
+            r#"
+use core::ops::ControlFlow;
+
+fn main() { ControlFlow::Break('\\\\') }
+"#,
+        );
```

Comment:
These outputs don't seem right to me, we effectively changed the string after the completion, simple escaping doesn't seem to work here. I'm not sure how we can solve this here though.

Reply:
Hmm, not sure if I'm following. I'm only escaping the text in the receiver position expression, which the user himself wrote and is expected to be rendered verbatim. Could I have an example where this doesn't work?

Reply:
Ah I think I got what you're saying. Yes, I'm turning `'\\'` into `'\\\\'` and sending it to the lsp client, which *then* unescapes every occurrence of `\\` to `\`. If we send `'\\'` without the escaping, the lsp client would interpret it as `'\'`, and that's what the original issue is about.

I understand the test fixture is confusing. Is there anything I can do to mitigate that?

Reply:
It might be good to add that to the code comment though for clarity.